src/keyevents.c

```diff
@@ -710,6 +710,8 @@ void feh_event_handle_generic(winwidget winwid, unsigned int state, KeySym keysy
 	}
 	else if (feh_is_kp(EVENT_toggle_pause, state, keysym, button)) {
 		slideshow_pause_toggle(winwid);
+		/* We need to reload the image to update the info string immediately. */
+		feh_reload_image(winwid, 0, 0);
```

Comment:
This causes feh to reset all zoom and pan options – in an interactive slideshow, where users may zoom and move the image, this is most likely not what they expect.

`winwidget_render_image` is more appropriate and more efficient here.